lib/config/presets/internal/group.ts

```diff
@@ -181,6 +182,15 @@ const staticGroups = {
       },
     ],
   },
+  phpstan: {
+    description: 'Group PHPStan packages together',
+    packageRules: [
+      {
+        matchPackagePatterns: ['phpstan'],
```

Comment:
```suggestion
        matchDatasources: ['packagist'],
        matchPackagePatterns: ['phpstan'],
```

This should be lmited to `packagist` (composer) packages. 

Shouldn't we limit to packages starting with `phpstan` instead allowing `foophpstanbar` ?
```suggestion
        matchDatasources: ['packagist'],
        matchPackagePatterns: ['^phpstan'],
```

Reply:
Starting with phpstan is problematic since all the third-party extensions won't start with it. But the syntax should be always `<vendor>/phpstan-<something>`, so I could do that.
Or we really limit it to starting with phpstan, then this group would be for phpstan and all official extensions only. Might also make sense.

Reply:
adapted to only include phpstan itself and extensions, what do you think? I also did a test-run, PR description is adapted